modules/developers-guide/partials/compare-orientations.adoc

```diff
@@ -0,0 +1,52 @@
+To compare the two approaches, let's look at an example that creates
+a container (table or type) to hold information about  a `reader`:
+
+[cols="l"]
+|===
+| CQL-oriented
+| reader: createTable(
+  keyspaceName:"library",
+  tableName:"reader",
+  partitionKeys: [ { name: "name", type: {basic: TEXT} } ]
+  # Secondary key used to access values within the partition
+  clusteringKeys: [ { name: "user_id", type: {basic: UUID}, order: "ASC" } ]
+  values: [
+    { name: "birthdate", type: {basic: DATE} }
+    { name: "email", type: {basic: SET, info:{ subTypes: [ { basic: TEXT } ] } } }
+    { name: "reviews", type: {basic: TUPLE,
+      info: { subTypes: [ { basic: TEXT }, { basic: INT }, { basic: DATE } ] } } }
+    { name: "addresses", type: { basic: LIST,
+      info: { subTypes: [ { basic: UDT, info: { name: "address_type", frozen: true } } ] } } }
+    ]
+  )
+}
+| GraphQL-oriented
```

Comment:
I understand what you mean now. "GraphQL-oriented GraphQL" is slightly awkward wording.

Reply:
By the way, "graphql-first" and "cql-first" will at least line up with what's in the code